modules/doc/content/newsletter/2021_02.md

```diff
@@ -1 +1,78 @@
 # MOOSE Newsletter (February 2021)
+
+## Incompressible Navier Stokes with Finite Volumes (INSFV)
+
+We now have a finite volume implementation of the incomprssible Navier-Stokes
+equations, with tests for lid-driven, natural convection, and channel
+flows. The implementation solves the mass and momentum conservation equations
+in a fully coupled manner, using a Rhie-Chow interpolation for the velocity
+field in order to avoid checker-board patterns in the pressure field (this
+checker-board pattern also arises
+in finite element calculations when using unstablized equal-order bases for the
+pressure and velocity). We have conducted verification tests in both 2D and 3D, checking for
+mass, momentum, and energy conservation (when the latter is included in the
+simulation). Moreover, through MMS testing the method is shown to be
+second-order convergent with respect to mesh refinement.
+
+## Default Automatic Differentiation (AD) configuration change
+
+We have transitioned MOOSE to use a globally-indexed sparse derivative container
+for automatic differentation by default. Experience has shown that this
+container type is hardly ever slower (and if it is, it is by a very small
+amount) and in some cases is much faster than the previous default AD container
+time (locally indexed non-sparse). This change in default configuration should
+hopefully be little-felt by users. However, for "power" AD developers, this
+change may open doors to more complex calculations. For example in finite volume
+gradient reconstruction, neighbor of neighbor information is required. Handling
+this larger element/cell stencil with a locally indexed container would be
+extremely difficult; however, it is very straightforward and natural using the
+globally indexed container. Here global vs. local means indexing based on
+globally numbered degrees of freedom vs. locally numbered degrees of freedom;
+one can hopefully imagine that as residuals become more and more dependent on non-local
+information, a local indexing scheme becomes more and more untenable.
+
+## GridPartitioner Improvement
+
+[GridPartitioner.md] creates a uniform processor grid that overlays the mesh to be partitioned.
+Then it assigns all elements of the to-be-partitioned mesh within each cell of the grid
+to the same processor. We used to explicitly create a native MOOSE mesh to represent
+the uniform processor grid. It caused issues for a specific scenario.  Recently, we
+improved this issue by creating a virtual grid instead of a native moose mesh.
+All operations for mesh element assignments are done using the virtual temporary grid only.
```

Comment:
```suggestion
[GridPartitioner.md] creates a uniform processor grid that overlays the mesh to be partitioned 
and then assigns all elements of the to-be-partitioned mesh within each cell of the grid
to the same processor. Previously, a native MOOSE mesh was explicitly created to represent
the uniform processor grid, and this could cause issues for a specific scenario.  This 
issue has been addressed by creating a virtual grid instead of a native moose mesh, and all 
operations for mesh element assignments are done using the virtual temporary grid only.
```
Here's a suggestion to re-work the wording here. Could you also elaborate (if possible) on that the `specific scenario` was that prompted this change? Or perhaps a link to the MOOSE issue and discussion that prompted it?

Reply:
@lindsayad Could you remind what is it?

Reply:
it interfered with attachment of geometric relationship managers

Reply:
```suggestion
[GridPartitioner.md] creates a uniform processor grid that overlays the mesh to be partitioned 
and then assigns all elements of the to-be-partitioned mesh within each cell of the grid
to the same processor. Previously, a native MOOSE mesh was explicitly created to represent
the uniform processor grid, and this could cause issues for a specific scenario where the native 
moose mesh interfered with attachment of geometric relationship managers.  This 
issue has been addressed by creating a virtual grid instead of a native moose mesh, and all 
operations for mesh element assignments are done using the virtual temporary grid only.
```

Reply:
@cticenhour This makes sense?

Reply:
I made a couple edits to the suggested change comment, and I think that's good to go. Thank you!

Reply:
Right, I can not commit my suggestion. We need someone to click button. Or you could create a fresh suggestion, and then I will be able to  click button.

Reply:
I'll create a fresh one, just a second.